crawler/file_handler.py

```
@@ -0,0 +1,179 @@
+
+def _extract_hwpx(path: Path) -> tuple[str, str]:
+    """HWPX(ZIP+XML) 에서 텍스트 추출."""
+    try:
+        texts = []
+        with zipfile.ZipFile(path) as z:
+            section_files = sorted(
+                [n for n in z.namelist() if n.startswith("Contents/section") and n.endswith(".xml")]
+            )
+            for section in section_files:
+                with z.open(section) as f:
+                    root = ET.parse(f).getroot()
```

Ruff (0.15.9)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)